### PR TITLE
Add an option to disable case-sensitive routing

### PR DESCRIPTION
**Pull request by mcollina**
See https://github.com/fastify/fastify/issues/1007.

@thearegee would this work for you?

### CODE REVIEW

**Comment by thearegee** [comment]
@mcollina this looks perfect, could just do with a note maybe in the README? 
Just so people know its available

**Comment by mcollina** [comment]
Done.

File: index.js [hunk 1]
```diff
@@ -308,6 +320,9 @@ Router.prototype.lookup = function lookup (req, res) {
 }
 
 Router.prototype.find = function find (method, path, version) {
+  if (!this.caseSensitive) {
```
Comment (delvedor) [inline index.js]: `if (this.caseSensitive === false)?`
It should make V8 happier (but I'm not super sure because the value is already a Boolean).

Comment (mcollina) [reply index.js]: updated.

**Comment by mcollina** [comment]
@delvedor updated.

**Pull request merged**